docs/content/getting-started.md

```diff
@@ -0,0 +1,260 @@
+
+:::
+
+A bot token is also needed to interact with the Web API methods as your app's bot user. We can gather this as follows:
+
+1. Navigate to the **OAuth & Permissions** on the left sidebar and install your app to your workspace to generate a token.
+2. After authorizing the installation, you'll return to the **OAuth & Permissions** page and find a **Bot User OAuth Access Token**:
+
+![OAuth Tokens](/img//bot-token.png "Bot OAuth Token")
+
+3. Copy the bot token beginning with `xoxb` from the **OAuth & Permissions page** and then store it in a new environment variable:
+
+```sh
+$ export SLACK_BOT_TOKEN=xoxb-example
+```
+
+After the app has been created, it is time to run it:
+
+```sh
+$ npm run start
+...
+[INFO]  bolt-app ⚡️ Bolt app is running!
+```
+
+With the app running, you can test it out with the following steps:
+
+1. Open a direct message with your app or invite the bot to a public channel.
+2. Send "hello" to the current conversation and wait for a response.
+3. Click the attached button labelled "Click me" to post another reply.
+
+After confirming the app responds, celebrate, then interrupt the process by pressing `CTRL+C` in the terminal to stop your app from running.
+
+</TabItem>
+</Tabs>
+
+## Updating the app
+
+At this point, you're successfully running the getting started Bolt for JavaScript [app](https://github.com/slack-samples/bolt-js-getting-started-app)!
+
+The defaults included leave opportunities abound, so to personalize this app let's now edit the code to respond with a kind farewell.
+
+#### Responding to a farewell
+
+Chat is a common thing apps do and responding to various types of messages can make conversations more interesting.
+
+Using an editor of choice, open the `app.js` file and add the following message listener after the "hello" handler:
+
+```js
+app.message('goodbye', async ({ say }) => {
+  const responses = ['Adios', 'Au revoir', 'Farewell'];
+  const parting = responses[Math.floor(Math.random() * responses.length)];
+  await say(`${parting}!`);
+});
+```
+
+Once the file is saved, restart your app to make sure the latest changes are being used, then go back to channel and say "goodbye".
+
+#### Customizing app settings
+
+The created app will have some placeholder values and a small set of [scopes](https://docs.slack.dev/reference/scopes) to start, but we recommend exploring the customizations possible on app settings.
+
+<Tabs groupId="cli-or-terminal">
+<TabItem value="cli" label="Slack CLI">
+
+Visit app settings for your app with the following command:
+
+```sh
+$ slack app settings
+```
+
+This will open the following page in a web browser:
+
+![Basic Information page](/img//basic-information-page.png "Basic Information page")
+
+</TabItem>
+<TabItem value="terminal" label="Browser">
+
+Return back to the pages of customization for your app at a similar URL:
+
+```
+https://api.slack.com/apps/A0123456789
+```
```

Comment:
suggestion: I feel like new developers will paste this URL and become confused around why it's a 404 page. Instead, let's hold their hand a little more.
```suggestion
Browse to https://api.slack.com/apps and select your app from the list.

This will open the following page:

![Basic Information page](/img//basic-information-page.png "Basic Information page")
```